public/app/features/provisioning/SyncRepository.tsx

```diff
@@ -30,28 +30,30 @@ export function ImportFromRepository({ repository }: Props) {
   });
 
   useEffect(() => {
+    // TODO... not true anymore --
+    // clicking sync just means we queued the job; can watch it until finished
```

Comment:
How do we know if the job has been successfully completed?

Reply:
the response will have a job "name" in the metadata -- we can either watch it, or poll until it completes.

The list of running jobs is currently shown in the repository status page